services/QuillLMS/package.json

```diff
@@ -38,28 +38,28 @@
     "rendering"
   ],
   "dependencies": {
+    "@originjs/vite-plugin-commonjs": "^1.0.3",
+    "@rollup/plugin-replace": "^5.0.2",
+    "@vitejs/plugin-react": "^3.1.0",
     "foreman": "^3.0.1",
     "lazysizes": "^5.2.0-beta1",
-    "puppeteer": "^19.5.2",
     "postcss": "^8.4.21",
     "postcss-import": "^15.1.0",
     "postcss-preset-env": "^8.0.1",
     "postcss-scss": "^4.0.6",
+    "puppeteer": "^19.5.2",
     "react-csv": "^1.1.2",
     "rollup-plugin-friendly-type-imports": "^1.0.3",
-    "sass": "^1.58.3",
-    "vite": "^4.5.2",
-    "vite-plugin-ruby": "^3.1.3",
-    "@originjs/vite-plugin-commonjs": "^1.0.3",
-    "@vitejs/plugin-react": "^3.1.0",
-    "@rollup/plugin-replace": "^5.0.2"
+    "sass": "^1.58.3"
   },
   "devDependencies": {
     "@types/enzyme": "^3.10.13",
     "@types/enzyme-adapter-react-16": "^1.0.6",
     "autoprefixer": "^10.4.13",
     "qs": "^6.9.0",
-    "redux-test-utils": "^0.2.2"
+    "redux-test-utils": "^0.2.2",
+    "vite": "^5.3.4",
+    "vite-plugin-ruby": "^5.0.0"
```

Comment:
The `vite upgrade` script moved `vite-plugin-ruby` into `devDependencies`. It should have been there all along